zerver/data_import/import_util.py

```diff
@@ -373,6 +372,10 @@ def build_realm(
         zerver_realmemoji=[],
         zerver_realmfilter=[],
         zerver_realmplayground=[],
+        zerver_realmauthenticationmethod=[
+            {"realm": realm_id, "name": name, "id": i}
+            for i, name in enumerate(all_implemented_backend_names(), start=1)
```

Comment:
This seems correct for say, Slack import. If I'm not mistaken, this is a case where we're not preserving whatever backends were enabled in the original configuration, right? Probably should be discussed and/or documented in the zulip=>zulip import docs.

Reply:
I guess one option would be to do this only if that table isn't present in the export.

Reply:
This is not used in the zulip=>zulip import, `build_realm`  is only for other tools - so this change is just replicating the current behavior of enabling all backends in those cases (currently this behavior happens through the default value of the bitfield)
```
zerver/data_import/gitter.py:    realm = build_realm(zerver_realm, realm_id, domain_name)
zerver/data_import/import_util.py:def build_realm(
zerver/data_import/mattermost.py:    realm = build_realm(zerver_realm, realm_id, domain_name)
zerver/data_import/rocketchat.py:    realm = build_realm(zerver_realm, realm_id, domain_name)
zerver/data_import/slack.py:    realm = build_realm(zerver_realm, realm_id, domain_name)
```

Reply:
OK; but maybe that behavior doesn't make sense; in theory the system where we're running this step might have different backends from the one where we're doing the import. We don't use that ability in practice, but probably we want to set the enabled backends at import time in some way? Probably worth a #backend discussion as a follow-up.

Reply:
I opened https://github.com/zulip/zulip/issues/25164 for figuring this out; I don't think it's a regression; but I don't think either the previous import logic here or the new import logic feels correct.